gremlin-go/driver/connection_test.go

```diff
@@ -656,6 +656,32 @@ func TestConnection(t *testing.T) {
 		resetGraph(t, g)
 	})
 
+	t.Run("Test Traversal. GetResultSet", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("Test Traversal.GetResultSet", func(t *testing.T) {
```

Reply:
Will edit locally along with other changes.

Reply:
Actually the period isn't meant to be a functional, just to follow the above Next/Has Next test name format. I'll just remove the period.